src/qt/trafficgraphwidget.cpp

```diff
@@ -47,13 +47,14 @@ int TrafficGraphWidget::getGraphRangeMins() const
 
 void TrafficGraphWidget::paintPath(QPainterPath &path, QQueue<float> &samples)
 {
-    int h = height() - YMARGIN * 2, w = width() - XMARGIN * 2;
-    int sampleCount = samples.size(), x = XMARGIN + w, y;
+    int sampleCount = samples.size();
     if(sampleCount > 0) {
+        int h = height() - YMARGIN * 2, w = width() - XMARGIN * 2;
+        int x = XMARGIN + w;
```

Comment:
Sorry for another nit, but why not inline this `x` (`YMARGIN + h` is also inlined`)? Then you can declare `int x` on line 56 (or inline those two as well).

Currently, there something like shadowing going on, where `x` is not modified inside the loop, as they are independent.